pkg/obfuscator/mac_address.go

```diff
@@ -1,15 +1,25 @@
 package obfuscator
 
 import (
+	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
 )
 
-const StaticMacReplacement = "xx:xx:xx:xx:xx:xx"
+const (
+	// staticMacReplacement refers to a static replacement for any identified MAC address.
+	staticMacReplacement = "xx:xx:xx:xx:xx:xx"
+	// consistentMACTemplate refers to a consistent replacement for any identified MAC address
+	consistentMACTemplate = "x-mac-%06d-x"
```

Comment:
Try to make this around as many characters as there are in a MAC address.

Reply:
Addressed